pages/dkp/kommander/2.2/licensing/enterprise/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout.pug
+navigationTitle: DKP Enterprise
+title: DKP Enterprise
```

Comment:
This looks really good!! I love the pictures. When we get into reorganization for Enterprise and Essential, we should use these images in the introduction to DKP section (to be created).